docs/src/apps/jsonrpc-api.md

```diff
@@ -929,6 +930,41 @@ curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","id":1, "m
 {"jsonrpc":"2.0","result":"ENvAW7JScgYq6o4zKZwewtkzzJgDzuJAFxYasvmEQdpS","id":1}
 ```
 
+### getStakeActivation
+
+Returns epoch activation information for a stake account
+
+#### Parameters:
+
+* `<string>` - Pubkey of stake account to query, as base-58 encoded string
+* `<object>` - (optional) Configuration object containing the following optional fields:
+  * (optional) [Commitment](jsonrpc-api.md#configuring-state-commitment)
+  * (optional) `epoch: <u64>` - epoch for which to calculate activation details. If parameter not provided, defaults to current epoch.
+
+#### Results:
+
+The result will be a JSON object with the following fields:
```

Comment:
This comment comes after my two "activating"/"deactivating" nits.   I think a overall "state" field would be very useful.  Right now the user needs to infer this based on the three fields. 

What about something like:
* `state: <string>` - the current state of the stake account: `inactive`, `activating`, `active`, `deactivating` (I don't love these state names but they're a start)
* `active: <u64>` - amount of stake that is currently active
* `inactive: <u64>` - amount of stake that is not active (eg, for a state account in the `activating` state, this will be the remaining stake that needs to warm up)

Reply:
Good idea. I suppose we could go to "warming up" and "cooling down" for state instead of activating/deactivating. Do those appeal any better? Would probably need to update the amount fields to match.

Reply:
@mvines , updated. I stuck with activating/deactivating for now. Thoughts?